calculators/helpers/ItemNetworthHelper.js

```diff
@@ -0,0 +1,212 @@
+     * @returns {boolean} Whether the item is cosmetic
+     */
+    isCosmetic() {
+        const testId = (this.itemId + this.itemName).toUpperCase();
+        const isSkinOrDye = testId.includes('DYE') || testId.includes('SKIN');
+        const isCosmetic = this.skyblockItem.category === 'COSMETIC' || this.itemLore.at(-1)?.includes('COSMETIC');
+
+        return isCosmetic || isSkinOrDye || this.isRune();
+    }
+
+    /**
+     * Checks if the item is recombobulated
+     * @returns {boolean} Whether the item is recombobulated
+     */
+    isRecombobulated() {
+        return this.extraAttributes.rarity_upgrades > 0 && !this.extraAttributes.item_tier;
+    }
+
+    /**
+     * Checks if the item is soulbound
+     * @returns {boolean} Whether the item is soulbound
+     */
+    isSoulbound() {
+        return !!(this.extraAttributes.donated_museum || this.itemLore.includes('§8§l* §8Co-op Soulbound §8§l*') || this.itemLore.includes('§8§l* §8Soulbound §8§l*'));
+    }
+
+    /**
+     * Sets the base price for the item
+     * @param {object} prices A prices object generated from the getPrices function
+     */
+    getBasePrice(prices) {
+        // Get the item id and name
+        this.itemName = this.getItemName();
+        this.itemId = this.getItemId(prices);
+
+        // Get the base price for the item
+        const itemPrice = prices[this.itemId] ?? 0;
+        this.price = itemPrice * this.itemData.Count;
+        this.base = itemPrice * this.itemData.Count;
+
+        // Check if the item has a skin - TODO: does the first if block in `getItemId` already do this?
+        if (this.extraAttributes.skin && !this.nonCosmetic) {
+            const newPrice = prices[`${this.baseItemId}_SKINNED_${this.extraAttributes.skin}`];
+            if (newPrice && newPrice > this.price) {
+                this.price = newPrice * this.itemData.Count;
+                this.base = newPrice * this.itemData.Count;
+            }
+        }
+
+        // Check if the item has a price paid attribute - TODO: might want to move to a handler?
+        if (!this.price && this.extraAttributes.price) {
+            this.price = parseInt(this.extraAttributes.price) * 0.85;
+            this.base = parseInt(this.extraAttributes.price) * 0.85;
+        }
+
+        // UPGRADABLE ARMOR PRICE CALCULATION (e.g. crimson)
+        if (!this.price) {
+            // if armor piece does not have base value
+            const prestige = prestiges[this.itemId.toUpperCase()];
+            if (prestige) {
+                for (const prestigeItem of prestige) {
+                    if (isNaN(this.price)) this.price = 0;
+                    if (this.skyblockItem?.upgrade_costs) this.price += starCosts(prices, this.calculation, this.skyblockItem.upgrade_costs, prestigeItem);
+                    if (this.skyblockItem?.prestige?.costs) this.price += starCosts(prices, this.calculation, this.skyblockItem.prestige.costs, prestigeItem);
+                }
+            }
+        }
```

Comment:
Should these be converted to handlers and the getBasePrice method runs a specific set of handlers that the calculator does not? Also noticed that god roll would be a base price, not added.